client/src/app/site/login/components/login-mask/login-mask.component.ts

```diff
@@ -191,6 +196,12 @@ export class LoginMaskComponent extends BaseComponent implements OnInit, OnDestr
         this.router.navigate(['./reset-password'], { relativeTo: this.route });
     }
 
+    private raiseHttpWarning(): void {
```

Comment:
I would expect something like "checkHttpConnection" or "checkForHttp" rather than "raiseHttpWarning". Reading "raise...Warning" suggests that a warning will always be shown - regardless whether a check succeed or not.